.github/workflows/build.yaml

```diff
@@ -13,6 +16,7 @@ env:
 jobs:
   build-crux:
     runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
```

Comment:
Never seen this before 🤔

Reply:
It avoids both `push` and `pull_request` triggering on PRs local to `redbadger/crux` and building twice. I didn't want to just disable `push` (or restrict it to just master), but that is the other option.

Reply:
Interesting, nice